### PR TITLE
Specify which toml file is being warned about

### PR DESCRIPTION
**Pull request by zisko**
There is a Cargo.toml file that can be confused in this warning.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1354,6 +1354,15 @@
...
       ]
+    },
+    {
+      "login": "tyirenkyi",
+      "name": "Timothy Yirenkyi",
+      "avatar_url": "https://avatars.githubusercontent.com/u/39198645?v=4",
+      "profile": "https://github.com/tyirenkyi",
+      "contributions": [
+        "doc"
+      ]
```
Comment (adilanchian) [inline .all-contributorsrc:1365]: You don't need to add this :). We have a bot that does all this automatically!

File: README.md [hunk 2]
```diff
@@ -274,6 +274,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="http://ertyurk.com"><img src="https://avatars.githubusercontent.com/u/30368068?v=4?s=100" width="100px;" alt=""/><br /><sub><b>ertyurk</b></sub></a><br /><a href="https://github.com/buildspace/buildspace-projects/commits?author=ertyurk" title="Documentation">📖</a></td>
     <td align="center"><a href="https://github.com/joshtsch"><img src="https://avatars.githubusercontent.com/u/14851820?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Joshua Tscheschlog &#124; Dev #3635</b></sub></a><br /><a href="https://github.com/buildspace/buildspace-projects/commits?author=joshtsch" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/tyirenkyi"><img src="https://avatars.githubusercontent.com/u/39198645?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Timothy Yirenkyi</b></sub></a><br /><a href="https://github.com/buildspace/buildspace-projects/commits?author=tyirenkyi" title="Documentation">📖</a></td>
```
Comment (adilanchian) [inline README.md:277]: Same here

**Review by adilanchian — CHANGES_REQUESTED** [review]
Core changes look good - 

Just remove your changes you made to the `README` && `.all-contributors` file :).

**Comment by adilanchian** [comment]
@zisko - friendly ping here to see if you are still wanting to do this :)

File: .all-contributorsrc (hunk 1 above)
Comment (zisko) [reply .all-contributorsrc]: so weird! I must have pulled in those changes accidentally- fixing now.

**Comment by zisko** [comment]
Fixed :)

**Review by adilanchian — APPROVED** [review]
Sorry for the late reply - 

Looks great! TY!

**Comment by adilanchian** [comment]
@all-contributors please add @zisko  for doc

**Pull request merged**